docs/quickstart-gke.md

```diff
@@ -173,8 +173,7 @@ BKPR creates and manages a Cloud DNS zone which is used to map external access t
 Query the name servers of the zone with the following command and configure the records with your domain registrar.
 
   ```bash
-  export GCLOUD_DNS_ZONE_NAME=bkpr-${GCLOUD_DNS_ZONE//./-}
-  export GCLOUD_DNS_ZONE_NAME=${GCLOUD_DNS_ZONE_NAME:0:30}
+  GCLOUD_DNS_ZONE_NAME=$(gcloud dns managed-zones list --filter dnsName:${GCLOUD_DNS_ZONE} --format='value(name)')
```

Comment:
+1, much nicer.  The previous commands were surprising (and thus distracting to the reader) unless you already knew the GCP details of how this name is formed.